test/integration/create_user_test.rb

```diff
@@ -1,14 +1,6 @@
-require 'test_helper'
-
-class CreateUserTest < ActionDispatch::IntegrationTest
-  before do
```

Comment:
There is likely much more that can go away, such as DatabaseCleaner and FactoryGirl, but I'm not quite there yet.